geoindicators/src/main/groovy/org/orbisgis/SpatialUnits.groovy

```diff
@@ -185,3 +187,40 @@ static IProcess createBlocks(){
             }
     )
 }
+
+/**
+ * This process is used to link each object of a lower scale to an upper scale ID (i.e.: building to RSU)
```

Comment:
maybe add information about the method. Because we use the max intersection area?

Reply:
you are right I do that.